src/main/webapp/scripts/index.js

```diff
@@ -6,4 +6,14 @@ $().ready(function() {
       eraseCookie('location');
     }
   });
+  $('#address').val(getCookie("address"));
+});
+
+$('#address').keypress(function(event){
+  let keycode = (event.keyCode ? event.keyCode : event.which);
+  // If enter key is pressed for address input, store the address as a cookie.
```

Comment:
[Optional] I'm not sure if waiting for the user to press enter is the best idea.

Reply:
Do you think it would be better to add another button?